lib/call-stack-to-nix.nix

```diff
@@ -14,6 +14,10 @@
 , stack-sha256 ? null
 , materialized ? null # Location of a materialized copy of the nix files
 , checkMaterialization ? null # If true the nix files will be generated used to check plan-sha256 and material
+, compiler-nix-name ? pkgs.haskell-nix.defaultCompilerNixNameTODO # We should get this from stack.yaml
```

Comment:
Not if we just use the same nix-tools all the time!

Reply:
Doing that now for stack projects.